x/upgrade/keeper/msg_server.go

```diff
@@ -10,12 +10,12 @@ import (
 )
 
 type msgServer struct {
-	Keeper
+	*Keeper
 }
 
 // NewMsgServerImpl returns an implementation of the upgrade MsgServer interface
 // for the provided Keeper.
-func NewMsgServerImpl(k Keeper) types.MsgServer {
+func NewMsgServerImpl(k *Keeper) types.MsgServer {
```

Comment:
Why is a reference needed?

Reply:
Because we want to set the version setter to the keeper here: https://github.com/cosmos/cosmos-sdk/pull/14216/files#diff-bde406b9030d9305e4d44025b556dae6d7c3dd3c924d28038d846910631a8a44R218-R220